tests/tasks/test_task_benchmarks.py

```diff
@@ -0,0 +1,75 @@
+import re
+
+import pytest
+import torch
+
+from sbibm import get_available_tasks, get_task
+from sbibm.algorithms.sbi.snpe import run
+from sbibm.metrics.ppc import median_distance
+
+# maybe use the pyro facilities
+torch.manual_seed(47)
```

Comment:
Yes, let's use `set_rng_seed` from `pyro.utils` instead which will also seed `numpy` and `random`

Reply:
done with 9e91818